src/screens/admin/AddFields.tsx

```
@@ -146,6 +148,7 @@ const AddFields: React.FC = () => {
 		if (!form.label.trim()) newErrors.label = 'Label is required';
 		if (!form.name.trim()) newErrors.name = 'Name is required';
 		if (!form.type) newErrors.type = 'Type is required';
+		if (form.ordering === '' || isNaN(Number(form.ordering))) newErrors.ordering = 'Ordering is required and must be a number';
```

ESLint

[error] Replace `·` with `⏎↹↹↹`

(prettier/prettier)